### PR TITLE
🐛 fix sending data as query on DELETE method

### PR DESCRIPTION
**Pull request by BHesseldieck**

### CODE REVIEW

File: packages/cli/src/UserManagement/routes/users.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 /* eslint-disable import/no-cycle */
 import { Response } from 'express';
+import { LoggerProxy as Logger } from 'n8n-workflow';
 import { In } from 'typeorm';
 import validator from 'validator';
-import { LoggerProxy as Logger } from 'n8n-workflow';
 
 import { Db, InternalHooksManager, ITelemetryUserDeletionData, ResponseHelper } from '../..';
-import { N8nApp, PublicUser } from '../Interfaces';
+import { SharedCredentials } from '../../databases/entities/SharedCredentials';
```
Comment (ivov) [inline packages/cli/src/UserManagement/routes/users.ts:9]: In future can we please avoid sorting imports? As it adds to noise to review in the PR. If we want this everywhere, let's add an ESLint rule and autofix.

**Review by ivov — COMMENTED** [review]
Manually tested on sqlite: workflow preserved, ownership switched. All tests passing in all three DBs.

To prevent this from happening again, in `DELETE /users/:id with transferId should perform transfer` we should add an expectation that the transferred workflow still exists, plus same for a transferred credential.

**Review by ivov — APPROVED** [review]
Thanks for fixing!

**Comment by janober** [comment]
Got released with `n8n@0.193.0`

**Pull request merged**